controller/pkg/counters/types.go

```diff
@@ -17,6 +17,7 @@ const (
 // CounterType custom counter error type.
 type CounterType int
 
+//go:generate stringer -type=CounterType -trimprefix Err
```

Comment:
Is there a `go generate` in a Makefile somewhere? Or do you have to run it manually in the future?

Reply:
Run it manually go generate will not walk vendor directory which is where the trireme-lib resides. 
Everytime we modufy counters we need to generate the countertypes_string.go file and check it in